src/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -78,6 +78,9 @@ Sample of usage in the product listing page layout:
 <block class="Magento\Catalog\Block\Product\ListProduct" name="category.products.list" as="product_list" template="Magento_Catalog::product/list.phtml"/>
 ```
 
+{:.bs-callout-warning}
+When define block if `template` attribute not available in layout, so that block will not render by default
```

Comment:
```suggestion
When define block if `template` attribute not available in layout (or via `setTemplate()` method in the block class), that block will not render by default
```

Reply:
This sentence does not scan well in English and is confusing, can you rephrase to make it clearer what it is trying to explain?

Reply:
In this scope of layout IMO we should not need mention with php way approach ?